packages/patternfly-4/react-core/src/components/Nav/NavToggle.js

```diff
@@ -20,7 +20,11 @@ class NavToggle extends React.Component {
   state = {
     value: this.props.defaultValue
   };
-
+  componentWillReceiveProps(nextProps) {
```

Comment:
this lifecycle method is soon to be deprecated. It works with our target version of React, but I recommend looking at `getDerivedStateFromProps` and `componentDidUpdate`.

https://reactjs.org/docs/react-component.html#unsafe_componentwillreceiveprops
https://egghead.io/lessons/react-refactor-componentwillreceiveprops-to-getderivedstatefromprops-in-react-16-3